spec/models/notification_spec.rb

```diff
@@ -371,4 +371,20 @@
       end.to change(Notification, :count).by(1)
     end
   end
+
+  describe "#remove_each_without_delay" do
```

Comment:
We will want to remove the `_without_delay` method soon, so it makes more sense to test `remove_each` using the `perform_enqueued_jobs` helper.

Reply:
Hello, I have done that.